PWGCF/Flow/Tasks/resonancesGfwFlow.cxx

```diff
@@ -271,10 +359,20 @@ struct ResonancesGfwFlow {
       minusdaug = ROOT::Math::PxPyPzMVector(partminus.px(), partminus.py(), partminus.pz(), plusmass);
       mom = plusdaug + minusdaug;
 
+      double pt = mom.Pt();
+      double invMass = mom.M();
```

Comment:
I would suggest using the methods in `Common/Core/RecoDecay.h` they are by far more efficient than using `PxPyPzMVector`